### PR TITLE
PHP 8.2 with dependencies + support Alpine 3.17

### PR DESCRIPTION
**Pull request by kalaputsik**
## PR Description
Tickets: 
- https://spryker.atlassian.net/browse/SC-13160 (PHP 8.2 support)
- https://spryker.atlassian.net/browse/SC-13299 (Alpine 3.17 support)

## Relation:
- https://github.com/spryker-sdk/docker-new-relic/pull/1 (image **spryker/newrelic-extension:10.9.0.324-php8.2** should be available)
- https://github.com/spryker/suite-nonsplit/pull/7765 (Nonsplit for QA)
- https://github.com/spryker/spryker/pull/10121 - (Core PR)
- https://github.com/spryker/docker-sdk/pull/323 - should be implemented

## Changelog:
- start support for Alpine 3.17
- start support for PHP 8.2
- update Blackfire: 1.86.4
- update Tideways: 5.5.14
- update NewRelic: 10.9.0.324
- update Composer: 2.5.5
- end of support for Alpine 3.14

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: alpine/3.16/8.2/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,195 @@
+# syntax = docker/dockerfile:1.0.2-experimental
+ARG SPRYKER_PHP_VERSION=8.2.5
+
+FROM spryker/newrelic-extension:10.9.0.324-php8.2 as spyker-newrelic-arm
+
+FROM php:${SPRYKER_PHP_VERSION}-fpm-alpine3.16
+
+ARG TARGETPLATFORM
+ARG COMPOSER_VERSION=2.5.5
```
Comment (zyuzka) [inline alpine/3.16/8.2/Dockerfile:9]: If it's possible - we should update versions into all our images

File: alpine/3.16/8.2/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    apcu \
+    redis"
+
+ARG ADDITIONAL_PHP_PECL_EXTENSIONS="\
+    pcov \
+    xdebug \
+    xhprof"
+
+ARG CFLAGS="-I/usr/src/php"
+RUN apk update \
```
Comment (zyuzka) [inline alpine/3.16/8.2/Dockerfile:77]: Please add next line:
`&& apk upgrade \`

File: debian/bullseye/8.2/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+    make \
+    netcat-openbsd \
+    openssh-client \
+    postgresql-client \
+    procps \
+    python3 \
+    unzip \
+    wget \
+    build-essential \
+    golang-go \
```
Comment (zyuzka) [inline debian/bullseye/8.2/Dockerfile:92]: I have a suggestion about Go in Debian. Please wait for my PR with vulnerability issues fixes

**Review by zyuzka — CHANGES_REQUESTED** [review]
What about other Alpine versions?

**Comment by alexanderM91** [comment]
Hi @kalaputsik, could you please remove Alpine 3.14 from our list as Alpine 3.14 will be deprecated in 6 days?

**Review by zyuzka — CHANGES_REQUESTED** [review]

**Review by alexanderM91 — APPROVED** [review]

**Pull request merged**